### ISSUE TITLE
[Feature] Video.js Rating Stars

### ISSUE THREAD

**Issue by elkorol** [stashapp/stash#3250]
**Is your feature request related to a problem? Please describe.**
Having to go into the edit panel to rate scenes is cumbersome, I end up tending to avoid rating scenes when in fact want to.

**Describe the solution you'd like**
I propose, a video.js plugin that superimposes rating stars onto the video player itself. Right aligned next to playback rate, there's a lot of space there that's without going to overboard unutilised (I have another idea for a video.js UI element, I'll add after this one). 

I think the colour scheme of the stars would be better in align with the current UI colour of other elements of the video player. So white stars

**Describe alternatives you've considered**
I've considered using injectable javascript to achieve the same but my knowledge of React is limited.

**Additional context**
<img width="1919" alt="485648" src="https://user-images.githubusercontent.com/57588838/206386819-f631a9b0-a553-4312-b584-83a237a27a47.png">

**Comment by skier233** [comment]
Would be nice. But would also need to work with other rating systems than the stars (ie decimal).

**Comment by StashPowerUser** [comment]
Being able to instantly rate scenes playing fullscreen would be excellent.

**Comment by DogmaDragon** [comment]
Would you consider this solved now that the ratings are available outside of Edit tab?

**Comment by StashPowerUser** [comment]
I would not until one click star rating is available in full screen.

**Comment by DogmaDragon** [comment]
@StashPowerUser you can use keyboard shortcuts for that https://docs.stashapp.cc/in-app-manual/keyboardshortcuts/#edit-scene-tab-shortcuts

**Comment by StashPowerUser** [comment]
Yeah, but the ratings aren't visible and the keyboard isn't always as convenient.

**Comment by cj12312021** [comment]
I would consider this issue completed. The star rating is no longer isolated to a specific tab, effectively addressing the pain point in the original comment. The source code is unlikely to place the rating directly in the video player. Since the rating is no longer isolated to a tab, it is much more flexible for users to relocate it via CSS or a plugin. This issue could be moved to the scripts repo if further work is requested.
